tests/exchange/test_exchange.py

```diff
@@ -4220,3 +4231,154 @@ def test_get_max_pair_stake_amount(
+    type(api_mock).has = PropertyMock(return_value={'fetchLeverageTiers': True})
+    default_conf['dry_run'] = False
+    default_conf['trading_mode'] = 'futures'
+    default_conf['margin_mode'] = 'isolated'
+
+    ccxt_exceptionhandlers(
+        mocker,
+        default_conf,
+        api_mock,
+        "binance",
+        "load_leverage_tiers",
+        "fetch_leverage_tiers",
+    )
+
+
+def test_parse_leverage_tier(mocker, default_conf):
+    exchange = get_patched_exchange(mocker, default_conf)
+
+    tier = {
+        "tier": 1,
+        "notionalFloor": 0,
+        "notionalCap": 100000,
+        "maintenanceMarginRatio": 0.025,
+        "maxLeverage": 20,
+        "info": {
+            "bracket": "1",
+            "initialLeverage": "20",
+            "notionalCap": "100000",
+            "notionalFloor": "0",
+            "maintMarginRatio": "0.025",
+            "cum": "0.0"
+        }
+    }
+
+    assert exchange.parse_leverage_tier(tier) == {
+        "min": 0,
+        "max": 100000,
+        "mmr": 0.025,
+        "lev": 20,
+        "maintAmt": 0.0,
+    }
+
+    tier2 = {
+        'tier': 1,
+        'notionalFloor': 0,
+        'notionalCap': 2000,
+        'maintenanceMarginRatio': 0.01,
+        'maxLeverage': 75,
+        'info': {
+            'baseMaxLoan': '',
+            'imr': '0.013',
+            'instId': '',
+            'maxLever': '75',
+            'maxSz': '2000',
+            'minSz': '0',
+            'mmr': '0.01',
+            'optMgnFactor': '0',
+            'quoteMaxLoan': '',
+            'tier': '1',
+            'uly': 'SHIB-USDT'
+        }
+    }
+
+    assert exchange.parse_leverage_tier(tier2) == {
+        'min': 0,
+        'max': 2000,
+        'mmr': 0.01,
+        'lev': 75,
+        "maintAmt": None,
+    }
+
+
+def test_get_maintenance_ratio_and_amt_exceptions(mocker, default_conf, leverage_tiers):
+    api_mock = MagicMock()
+    default_conf['trading_mode'] = 'futures'
+    default_conf['margin_mode'] = 'isolated'
+    mocker.patch('freqtrade.exchange.Exchange.exchange_has', return_value=True)
+    exchange = get_patched_exchange(mocker, default_conf, api_mock)
+
+    exchange._leverage_tiers = leverage_tiers
+    with pytest.raises(
+        OperationalException,
+        match='nominal value can not be lower than 0',
+    ):
+        exchange.get_maintenance_ratio_and_amt('1000SHIB/USDT', -1)
+
+    exchange._leverage_tiers = {}
+
+    with pytest.raises(
+        InvalidOrderException,
+        match="Maintenance margin rate for 1000SHIB/USDT is unavailable for",
+    ):
+        exchange.get_maintenance_ratio_and_amt('1000SHIB/USDT', 10000)
+
+
+@ pytest.mark.parametrize('pair,value,mmr,maintAmt', [
+    ('ADA/BUSD', 500, 0.025, 0.0),
+    ('ADA/BUSD', 20000000, 0.5, 1527500.0),
+    ('ZEC/USDT', 500, 0.01, 0.0),
+    ('ZEC/USDT', 20000000, 0.5, 654500.0),
+])
+def test_get_maintenance_ratio_and_amt(
+    mocker,
+    default_conf,
+    leverage_tiers,
```

Comment:
Is `leveraged_tiers` not needed? if it's not - i'd remove it to keep tests clean